features/video-recorder/popup.html

```diff
@@ -0,0 +1,53 @@
+<div class="ReactModalPortal STE-ReactModalPortal">
+    <div class="ReactModal__Overlay ReactModal__Overlay--after-open modal_modal-overlay_1Lcbx">
+        <div class="ReactModal__Content ReactModal__Content--after-open modal_modal-content_1h3ll prompt_modal-content_1BfWj"
+            tabindex="-1" role="dialog" aria-label="Rename Variable">
+            <div class="box_box_2jjDp" dir="ltr" style="flex-direction: column; flex-grow: 1;">
+                <div class="modal_header_1h7ps">
+                    <div class="modal_header-item_2zQTd modal_header-item-title_tLOU5">Video Recording</div>
+                    <div class="modal_header-item_2zQTd modal_header-item-close_2XDeL">
+                        <div aria-label="Close" class="close-button_close-button_lOp2G close-button_large_2oadS"
+                            role="button" tabindex="0"><img class="close-button_close-icon_HBCuO"
+                                src="data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA3LjQ4IDcuNDgiPjxkZWZzPjxzdHlsZT4uY2xzLTF7ZmlsbDpub25lO3N0cm9rZTojZmZmO3N0cm9rZS1saW5lY2FwOnJvdW5kO3N0cm9rZS1saW5lam9pbjpyb3VuZDtzdHJva2Utd2lkdGg6MnB4O308L3N0eWxlPjwvZGVmcz48dGl0bGU+aWNvbi0tYWRkPC90aXRsZT48bGluZSBjbGFzcz0iY2xzLTEiIHgxPSIzLjc0IiB5MT0iNi40OCIgeDI9IjMuNzQiIHkyPSIxIi8+PGxpbmUgY2xhc3M9ImNscy0xIiB4MT0iMSIgeTE9IjMuNzQiIHgyPSI2LjQ4IiB5Mj0iMy43NCIvPjwvc3ZnPg==">
+                        </div>
+                    </div>
+                </div>
+                <style>
```

Comment:
Is there any reason why you use inline styles?

Reply:
Mostly because of convenience